views/printerManagement.ejs

```diff
@@ -564,32 +564,31 @@
                     <div class="row">
                         <div class="col-12 col-sm-12 col-md-12 col-lg-2 col-xl-2">
                             <div class="list-group" id="list-tab" role="tablist">
-                                <a class="list-group-item list-group-item-action active" id="printer-connection-btn"
+                                <a class="list-group-item list-group-item-action active notyet" id="printer-connection-btn"
                                    data-toggle="list" href="#printer-connection" role="tab" aria-controls="messages"><i
-                                            class="fas fa-plug"></i> Connection</a>
-                                <a class="list-group-item list-group-item-action" id="printer-cost-btn"
+                                            class="fas fa-plug"></i> Connection <i class="fas fa-spinner fa-spin"></i></a>
+                                <a class="list-group-item list-group-item-action notyet" id="printer-cost-btn"
                                    data-toggle="list" href="#printer-cost" role="tab" aria-controls="messages"><i
-                                            class="fas fa-dollar-sign"></i> Cost</a>
-                                <a class="list-group-item list-group-item-action" id="printer-profile-btn"
+                                            class="fas fa-dollar-sign"></i> Cost <i class="fas fa-spinner fa-spin"></i></a>
+                                <a class="list-group-item list-group-item-action notyet" id="printer-profile-btn"
                                    data-toggle="list" href="#printer-profile" role="tab" aria-controls="messages"><i
-                                            class="fas fa-tags"></i> Profile</a>
-                                <a class="list-group-item list-group-item-action" id="printer-power-btn"
+                                            class="fas fa-tags"></i> Profile <i class="fas fa-spinner fa-spin"></i></a>
+                                <a class="list-group-item list-group-item-action notyet" id="printer-power-btn"
```

Comment:
notyet is a specific class I created to disable the action on a links. It's removed from here once the section has loaded. It's handy if you want to disable an "a" link as they don't work with disabled as buttons do.